src/graphql/__tests__/apolloServer.test.ts

```diff
@@ -33,15 +27,10 @@ import {
   DomainDoc,
   SuggestionDoc,
   TaskDoc,
-  TokenDoc,
```

Comment:
Why was this removed? Is the `TokenDoc` not used in tests anymore?

Reply:
That's right! `TokenDoc` is not used in the tests, even with the parts I've removed included